### DISCUSSION TITLE
Dirigente alias

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#177] [Idee e proposte]
Sarebbe possibile far modo che anche il dirigente/preside possa effettuare l'alias degli utenti dell'istituto ?

**Comment by trinko** [comment]
Personalmente ritengo che sia inopportuno. 
L'alias è pensato per l'amministratore in modo che possa risolvere eventuali problemi in modo rapido: un uso da parte del dirigente può scatenare contenziosi... con l'alias si può modificare qualsiasi cosa (es. voti), facendola risultare fatta dal docente stesso. Naturalmente nel sistema viene registrato chi ha effettuato realmente la modifica, ma non è una informazione visibile direttamente agli utenti.
